homeassistant/components/imap/sensor.py

```diff
@@ -13,6 +17,8 @@
 from . import ImapPollingDataUpdateCoordinator, ImapPushDataUpdateCoordinator
 from .const import DOMAIN
 
+IMAP_MAIL_COUNT = "imap_mail_count"
```

Comment:
We don't need a constant for this string as it's only used in the entity description.